lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
Collection returns an object as well that has a data key that is an array but the collection by itself is object.


> On Apr 4, 2016, at 9:56 PM, Adam Fenton <notifications@github.com> wrote:
> 
> —